types/accesscontrol/access_operation_map.go

```diff
@@ -0,0 +1,23 @@
+package accesscontrol
+
+// Alias for Map of MessageIndex -> AccessOperation -> Channel
+type MessageAccessOpsChannelMapping = map[int]AccessOpsChannelMapping
+
+// Alias for Map of AccessOperation -> Channel
+type AccessOpsChannelMapping = map[*AccessOperation][]chan interface{}
```

Comment:
Why is this definition a map on struct pointer? I thought we were keying on the struct itself previously?

Reply:
Sorry, forgot to include the context.go change, but it was using pointer as the key. It shouldn't though as we may need to construct it later on if we want more granular access ops

https://github.com/sei-protocol/sei-cosmos/pull/29/files#diff-10fabc84ecc74a1e1f8383ab56fb83d48a5528658a27ee5373d05a13d294d438L54